frontend/src/components/VerifyAccountCard/VerifyAccountCard.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Button, CardContent, Grid, Paper, Typography } from '@material-ui/core';
+import confirmEmailDrawing from '../../assets/VerifyAccountPage/ill-confirmation.svg';
```

Comment:
Let's make our imports cohesive. In my branch I have started importing everything from `index.js` files, so maybe it would be best for you to merge my PR branch into yours and change this import to use `index.js`

Reply:
My bad, it would be possible to write only `from '../../assets` and it should work fine.